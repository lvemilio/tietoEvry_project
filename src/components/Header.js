import React from 'react'

const header = ({title}) => {
    return (
        <header className = 'header'>
            <h1 style = {{textAlign:'center'}}>{title}</h1>
        </header>
    )
}


header.defaultProps = {
    title:'Triangle App'
}




export default header
