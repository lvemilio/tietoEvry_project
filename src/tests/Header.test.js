import React from 'react';
import {render,cleanup} from '@testing-library/react'
import "@testing-library/jest-dom"
import Header from '../components/Header'

afterEach(cleanup);

it("renders",()=> {
    const {asFragment} = render(<Header text = "Triangle App" />)
    expect(asFragment()).toMatchSnapshot();
});

