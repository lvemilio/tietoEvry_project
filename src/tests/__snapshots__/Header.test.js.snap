// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <header
    class="header"
  >
    <h1
      style="text-align: center;"
    >
      Triangle App
    </h1>
  </header>
</DocumentFragment>
`;
